src/utils/DialogUtils.ts

```diff
@@ -185,3 +186,15 @@ export function getTeamId(event: builder.IEvent): string {
     }
     return "";
 }
+
+// Generate the team info data in table format
+export function generateTableForTeamInfo(teamDetails: teams.TeamInfo): string {
```

Comment:
We only use this in FetchTeamInfoDialog, let's keep it there.

Reply:
Done.